executor/src/witgen/jit/function_cache.rs

```diff
@@ -44,8 +45,39 @@ pub struct FunctionCache<'a, T: FieldElement> {
     parts: MachineParts<'a, T>,
 }
 
+enum WitgenFunction<T: FieldElement> {
+    // TODO We might want to pass arguments as direct function parameters
```

Comment:
Can you move this comment back to `CompiledFunction`? It mainly concerns the WitgenFunctionParameters, i.e. could also be moved there.

Reply:
right!